pontoon/api/schema.py

```diff
@@ -45,9 +43,8 @@ class Locale(DjangoObjectType, Stats):
     class Meta:
         model = LocaleModel
         only_fields = (
-            'name', 'code', 'direction', 'script', 'population',
-            'total_strings', 'approved_strings', 'translated_strings',
-            'fuzzy_strings'
+            'name', 'code', 'direction', 'script', 'population', 'total_strings',
+            'approved_strings', 'fuzzy_strings'
```

Comment:
Nit, and coming after the battle, but since those are going to evolve over time, I'd recommend we put one field per line, sorted alphabetically. It's going to make following changes much much easier to read.